src/api/db/migrate/20161122121211_move_comments_data.rb

```diff
@@ -0,0 +1,16 @@
+class MoveCommentsData < ActiveRecord::Migration[5.0]
+  def change
+    ActiveRecord::Base.transaction do
+      Comment.inheritance_column = nil
+      Comment.find_each(batch_size: 5000) do |comment|
+        /\AComment(?<type>.+)/ =~ comment.type
+        raise "Uncorrected type for comment with id #{comment.id}" unless ['Project', 'Package', 'BsRequest'].include? type
```

Comment:
That's not true, it is inside a transaction. If one of them fails nothing will be saved in the database.

It shouldn't happens, but in case that there is any error we won't move wrong data.